src/components/Explore.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <div class="explore-wrapper">
+    <h2>Explore</h2>
+    <div class="explore-body">
+      <ul id="explore-list">
+        <li v-for="item in items">
+          <a v-bind:href="item.msg">{{item.msg}}</a>
```

Comment:
https://vuejs.org/v2/api/#v-bind

Shorthand form would be `:href="item.msg"`, it's a tad easier to type ^^.